integrations/pytorch-lightning/Neptune-PyTorch-Lightning-advanced.ipynb

```diff
@@ -195,7 +195,7 @@
     "                          'decay_factor': 0.95}\n",
     "\n",
     "LightningDataModule_Params = {'batch_size': 32,\n",
-    "                              'num_workers': 4,\n",
+    "                              'num_workers': 0,\n",
```

Comment:
I followed the advice here https://github.com/pytorch/pytorch/issues/5301 and it fixed Lightning with Python 3.8 on a Mac, but not sure if performance is as good.

I hope this is fine - we want to CI to pass, even if it takes 2 minutes longer. Plus this is not Neptune's problem but Lightning's.